static/js/HitchhikerJS.js

```diff
@@ -35,8 +35,9 @@ import RuntimeConfigReceiver from './runtime-config-receiver';
 export { RuntimeConfigReceiver };
 
 import RuntimeConfig from './runtime-config';
-window.AnswersExperience = {
-  runtimeConfig: new RuntimeConfig()
-};
+import AnswersExperience from './answers-experience';
+const runtimeConfig = new RuntimeConfig();
+const answersExperience = new AnswersExperience(runtimeConfig);
+window.AnswersExperience = answersExperience;
```

Comment:
nit: we could just have `window.AnswersExperience = new AnswersExperience(runtimeConfig);`